client/admin/interface.go

```diff
@@ -34,6 +34,7 @@ import (
 //go:generate gowrap gen -g -p . -i Client -t ../templates/errorinjectors.tmpl -o ../wrappers/errorinjectors/admin_generated.go -v client=Admin
 //go:generate gowrap gen -g -p . -i Client -t ../templates/grpc.tmpl -o ../wrappers/grpc/admin_generated.go -v client=Admin -v package=adminv1 -v path=github.com/uber/cadence-idl/go/proto/admin/v1 -v prefix=Admin
 //go:generate gowrap gen -g -p . -i Client -t ../templates/thrift.tmpl -o ../wrappers/thrift/admin_generated.go -v client=Admin -v prefix=Admin
+//go:generate gowrap gen -g -p . -i Client -t ../templates/tchannel.tmpl -o ../wrappers/tchannel/admin_generated.go -v client=Admin
```

Comment:
Could you elaborate on why it is tchannel? :)
As far as I understand, it is an abstraction around thrift or grpc clients.
Not sure what would be the best name here.
timed? timed_out?

Reply:
The naming was based off of this [comment](https://github.com/uber/cadence/blob/7fddc8984345048edb9f444411a9c0524ad4a4bf/client/frontend/client.go#L47)  under the constructors. However this is old code maybe previously used as a tchannel wrapper. I like timed better

Reply:
As far as I see @shijiesheng called it timeout wrapper in https://github.com/uber/cadence/pull/5728
Lets rename it to timeout then.

Reply:
done